tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -212,6 +237,9 @@ def get_supervised(builder):
 def get_citation(builder):
   return builder.info.citation
 
+def get_figure(builder):
+  return builder.info.full_name
```

Comment:
This should returns the same constant if none of the images exists. Otherwise, it will create multiple empty sections.

Reply:
Sorry for trouble but I don't get you properly what that means `This should returns the same constant if none of the images exists.` . Like when dataset is of text ?

Reply:
Currently, if the dataset do not have visualization, it will create multiple empty sections:

```
# Dataset
## Config0
* Figure: No figure
## Config1
* Figure: No figure
```
With this change:
```py
def get_figure(builder):
  return builder.info.full_name if _visualization_exists(builder) else None
```
Only one section gets created, which avoid redundancy:
```
# Dataset
* Figure: No figure
## Config0
## Config1
```

Reply:
Yes its create multiple FIgure: No figure available I will make changes thanks